src/components/Modals/CartModal.js

```diff
@@ -0,0 +1,33 @@
+import {
+  BackGround,
+  ModalContainer,
+  ConfirmRow,
+  ClearAll,
+  RemainAll,
+  ModalTitle,
+  ModalBody,
+} from "./styled";
+
+const CartModal = ({ showModal, deleteAll, noChange }) => {
```

Comment:
to make this modal more generic, u can have these as props:
showModal
onOk
onCancel
title
body
okText -> default value: "Ok"
cancelText -> default value: "Cancel"